test/hs/Test/Ganeti/HTools/Node.hs

```diff
@@ -134,6 +134,7 @@ genEmptyOnlineNode =
                        , Node.rMemForth = 0
                        , Node.pRem = 0
                        , Node.pRemForth = 0
+                       , Node.xMem = 0
```

Comment:
> xMem would change without needing any update, right?

But we don't want this to change. The name 'missing' is ambigous here (to me it was confusing at least at the beginning). It isn't the memory that is missing in order to accomodate something. It is the memory that went missing - it is being used by some unknown process on the node. During placements we assume that this memory doesn't change as we shuffle instances.